ledfx/effects/audio.py

```diff
@@ -792,10 +794,14 @@ def activate(self, channel):
             )
 
         self.audio = self._ledfx.audio
-        self._ledfx.audio.subscribe(self._audio_data_updated)
+        self.audio.subscribe(self._audio_data_updated)
+
+    def activate(self, channel):
+        _LOGGER.debug("Activating AudioReactiveEffect base class.")
+        super().activate(channel)
 
     def deactivate(self):
-        _LOGGER.info("Deactivating AudioReactiveEffect.")
+        _LOGGER.debug("Deactivating AudioReactiveEffect base class.")
         if self.audio:
```

Comment:
May actually be worth removing this completely, since self.audio should *always* exist now.